external/contracts/set/BasicIssuanceModule.sol

```diff
@@ -200,35 +242,70 @@ contract BasicIssuanceModule is ModuleBase, ReentrancyGuard {
      * @return address[]            List of component addresses
      * @return uint256[]            List of component units required to issue the quantity of SetTokens
      */
-    function getRequiredComponentUnitsForIssue(
+    function getRequiredComponentIssuanceUnits(
```

Comment:
This is nice. Having different names for this method between BIM and DIM was really annoying before :+1

Reply:
just noticed that they also differ in returned values... 
ideally BIMv2 would have to return an empty array as return param 3 (for debt positions) I think